data/wb.json

```diff
@@ -1,64 +1,76 @@
 {
-    "creators": [
-        {
-            "title": "Law of physics",
-            "body": "This is my first creator post",
-            "author": "Micheal Faraday",  
-            "id": 1
-        },
-        {
-            "title": "Mathematics",
-            "body": "This is my second creator post",
-            "author": "John fresco",  
-            "id": 2
-        },
-        {
-            "title": "English",
-            "body": "This is my third creator post",
-            "author": "Elizabeth florish",  
-            "id": 3
-        },
-        {
-            "title": "Biology",
-            "body": "This is my fourth creator post",
-            "author": "Paul Doe",  
-            "id": 4
-        },
-        {
-            "title": "Computer Science",
-            "body": "This is my fifth creator post",
-            "author": "Damilare Isreal",  
-            "id": 5
-        },
-        {
-            "title": "Business Admin",
-            "body": "This is my sixth creator post",
-            "author": "Yetunde Oloye",  
-            "id": 6
-        },
-        {
-            "title": "Foreign Exchange",
-            "body": "This is my seventh creator post",
-            "author": "Olawale  Salami",  
-            "id": 7
-        },
-        {
-            "title": "Anatomy",
-            "body": "This is my eighth creator post",
-            "author": "Ayomide Abiodun",  
-            "id": 8
-        },
-        {
-            "title": "Transport Management",
-            "body": "This is my ninth creator post",
-            "author": "Tobi Gabriel",  
-            "id": 9
-        },
-        {
-            "title": "Lab experiment",
-            "body": "This is my tenth creator post",
-            "author": "Dayo Ayodele",  
-            "id": 10
-        }
-    ]    
+  "creators": [
+    {
+      "title": "Law of physics",
+      "body": "This is my first creator post",
+      "author": "Micheal Faraday",
+      "id": "1"
+    },
+    {
+      "title": "Mathematics",
+      "body": "This is my second creator post",
+      "author": "John fresco",
+      "id": "2"
+    },
+    {
+      "title": "English",
+      "body": "This is my third creator post",
+      "author": "Elizabeth florish",
+      "id": "3"
+    },
```

Comment:
The ids in your json file are currently in string format. Kindly check the message i created on the whatsapp group to fix this.

Reply:
I have repeated the process several times and up till now it keeps bringing string after I have removed them.